trillian/ctfe/configpb/config.proto

```diff
@@ -101,6 +101,15 @@ message LogConfig {
   // log's key and can't sign STHs. Consequently, the log operator must ensure
   // to channel source log's STHs into CTFE.
   bool is_mirror = 12;
+
+  // The Maximum Merge Delay (MMD) of this log in seconds. See RFC6962 section 3
+  // for definition of MMD. If zero, the log does not provide an MMD guarantee.
+  int32 max_merge_delay_sec = 14;
```

Comment:
Does zero include the frozen case? Need to be able to construct the alert so that it doesn't fire for frozen logs. I think that's effectively `merge_delay_sec > expected_merge_delay_sec && max_merge_delay_sec > 0`

Reply:
I think the frozen case falls to `If zero, the log does not provide an MMD guarantee.` case, but I extended the comment.